main.js

```diff
@@ -11,10 +12,29 @@ if (PapanUtils.isElectron()) {
 const mainNode = require('./src/server/main-node.js')
 mainNode.main()
 
+function readJSON (filename) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(filename, (err, data) => {
+      if (err) resolve({})
+      resolve(JSON.parse(data))
+    })
+  })
+}
+
 if (argv.auth_server) {
-  let config = {}
-  const express = require('express')
-  const papanAuth = require('./src/server/auth/server.js')
-  let app = express()
-  papanAuth.registerServer(app, config).then(app.listen(8081))
+  let googleAuthConfig = {}
+  readJSON('config/google-auth-config.json').then((googleAuthConfigRead) => {
+    googleAuthConfig = googleAuthConfigRead
+    return readJSON('config/pg-config.json')
+  }).then((pgConfig) => {
```

Comment:
I don't think you want to do two consecutive promise resolutions here. `Promise.all` lets you resolve two promises in parallel, and I think that simplifies the code here by letting you eliminate the broader-scoped `googleAuthConfig` variable, and you only need one `then` call.